src-tauri/src/main.rs

```diff
@@ -36,8 +36,16 @@ fn main() {
     let tray = create_tray(&manager_state);
     tauri::Builder::default()
         .setup(|_app| {
-            // TODO: build the web assets as part of the build process, instead of assuming they are there
-            let asset_path = PathBuf::from("../../aw-webui/dist");
+            let webui_var = std::env::var("AW_WEBUI_DIR");
+            let webui_path = if let Ok(var_path) = &webui_var {
+                std::path::Path::new(var_path)
+            } else {
+                let path = std::path::Path::new("../aw-webui/dist");
+                println!("cargo:rustc-env=AW_WEBUI_DIR={}", path.display());
```

Comment:
This is only applicable to the build.rs script, setting it here does nothing (rustc has already compiled the project).

We should pass the AW_WEBUI_DIR from the Makefile (`AW_WEBUI_DIR=... cargo build`, or whatever) so that it's picked up when aw-server-rust is built, and that should be enough.

Those assets will then get bundled into the executable, and no override webui path should be passed when starting the server.